docs/can-guides/contribute/including-new-modules.md

```diff
@@ -0,0 +1,111 @@
+@page guides/contributing/include-new-modules Including New Modules
```

Comment:
This should probably be `Adding Ecosystem Packages` as that will be by far the most common.